### DISCUSSION TITLE
Speech Attendant

### DISCUSSION THREAD

**Discussion by tomlynn** [toverainc/willow-inference-server#110] [General]
I'm interested in learning more about the issues I'd encounter attempting to use WIS in an enterprise-wide speech attendant.  Initially it would handle the needs of internal employees and relieve them from trying to find a persons location/extension.  Pick up the phone, hit a key and connect to the speech attendant.  

I've watched Kristian's youtube videos and the interview with Luca on Clue Con Weekly.  All great stuff.  My assumption is I'd connect via SIP to WIS.  In the call with Luca Kristian mentions SIP capability, but I'm unsure if he meant Willow or WIS in that regard.  Hoping for WIS.  The easy path would be if WIS can respond 302 Temporarily moved to my session manager to transfer a call, but I'm guessing I need an IVR/Freeswitch to front end the call. 

And then there's a whole realm of training (continually) against existing directories.

At least there's no wake words!

Any advice is appreciated.  Too bad my home lab isn't up to the task.

**Comment by kristiankielhofner** [comment]
I talked about this a bit with Luca.

WIS doesn't support SIP natively and almost certainly never will. You would need to use FreeSWITCH to bridge to WIS either via WebRTC or buffering of audio and HTTP POST to WIS.
